src/Microsoft.Management.Deployment.Projection/ClassesDefinition.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Management.Deployment.Projection
+{
+    using System;
+    using System.Collections.Generic;
+
+    internal static class ClassesDefinition
+    {
+        private static Dictionary<Type, ClassModel> Classes { get; set; } = new()
+        {
+            [typeof(PackageManager)] = new()
+            {
+                ProjectedClassType = typeof(PackageManager),
+                InterfaceType = typeof(IPackageManager),
+                Clsids = new Dictionary<ClsidContext, Guid>()
+                {
+                    [ClsidContext.InProc] = new Guid("2DDE4456-64D9-4673-8F7E-A4F19A2E6CC3"),
```

Comment:
Not a fan of these being defined a second time, although it is certainly better than defining them for a 50th time in every caller.